python/benchmarks/bench.py

```diff
@@ -0,0 +1,48 @@
+import multiprocessing as mp
```

Comment:
Do you plan to check this in? 

If so, lets rename the file to `oxford_pet.py`